src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1938,6 +1938,71 @@ private void queryLoans(int limit)
     });
   }
 
+  @Test
+  public void canGetLoanWithPatronGroupAfterAnonymize() throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    InventoryItemResource item = itemsFixture.basedUponSmallAngryPlanet();
+    IndividualResource jessica = usersFixture.jessica();
+
+    IndividualResource individualResource = loansFixture.checkOutByBarcode(item,
+      jessica, new DateTime(2018, 4, 21, 11, 21, 43, DateTimeZone.UTC));
+
+    JsonObject savedLoan = loansStorageClient.get(individualResource.getId())
+      .getResponse().getJson();
+
+     savedLoan.remove("userId");
+
+     loansStorageClient.replace(individualResource.getId(), savedLoan);
+
+     JsonObject loan = loansClient.get(individualResource.getId()).getJson();
+
+     loanHasPatronGroupProperties(loan, "Regular Group");
+  }
+
+  @Test
+  public void canGetMultipleLoansWithPatronGroupAfterAnonymize() throws InterruptedException,
```

Comment:
I think this test could be used more generally for anonymized loans
```suggestion
  public void canGetMultipleAnonymizedLoans() throws InterruptedException,
```

Reply:
Sure